easybuild/easyconfigs/p/Python/Python-2.7.14-GCCcore-7.2.0-bare.eb

```diff
@@ -0,0 +1,37 @@
+name = 'Python'
+version = '2.7.14'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TAR_XZ]
```

Comment:
why not stick to `SOURCE_TGZ`?

Reply:
@rubendibattista unresolved, I see no reason to switch to `SOURCE_TAR_XZ`? You'll also need to change the checksum to `304c9b202ea6fbd0a4a8e0ad3733715fbd4749f2204a9173a58ec53c32ea73e8` if you switch back to `SOURCE_TGZ`